### DISCUSSION TITLE
Bed tramming wizard complaining about probe being out of bounds on front right point

### DISCUSSION THREAD

**Discussion by zurgeg** [mriscoc/Ender3V2S1#1064] [Q&A] [answered]
When I run the tramming wizard from the toolbar, it succeeds in probing the front left point, and then goes to the front right, touches the bed, and retracts immediately. After retracting, the probe (CR Touch) turns purple, and `Tramming error` displays on the screen with text at the bottom saying `Probe out of bounds`

Does anyone know what's happening?

**Comment by uiJLKJ** [comment]
Hi, have you tried a manual tramming ? I would suggest you to take a piece of paper and then do it, your bed may be slightly out of touch for the probing. This happened to me once I had to higher the bed a little.

**Reply by zurgeg** [reply]
Just tried that, still out of bounds

**Reply by PipeyMnM** [reply]
Reset the mesh/ load it/ default it

**Reply by mriscoc** [reply]
Follow the steps here: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides

**Comment by mriscoc** [comment, marked as answer]
https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides

**Comment by Livebaitdrifter** [comment]
Ive had this with a og minimus cooler, shimming the blt lower stopped it. 
Today it came back after a year.
No settings will stop this error.
